website/src/components/GridPattern.tsx

```diff
@@ -0,0 +1,24 @@
+import { useId } from 'react'
+
+export function GridPattern(
+  props: Omit<React.ComponentPropsWithoutRef<'pattern'>, 'id'>,
+) {
+  let patternId = useId()
+
+  return (
+    <svg aria-hidden="true" className="absolute inset-0 h-full w-full">
```

Comment:
**polish**: It is not the responsibility of this component to hold the absolute position in my opinion. Especially since its parents are already in absolute position.

```suggestion
    <svg aria-hidden="true" className="inset-0 h-full w-full">
```